OpenRA.Mods.Common/Traits/Minelayer.cs

```diff
@@ -63,6 +63,12 @@ public class MinelayerInfo : TraitInfo, Requires<RearmableInfo>
 		[Desc("Ammo the minelayer consumes per mine.")]
 		public readonly int AmmoUsage = 1;
 
+		[Desc("Number of ticks it takes to lay a mine. Zero means mine-laying is instantaneous.")]
+		public readonly int PreLayDelay = 0;
+
+		[Desc("Delay in ticks after mine is laid before minelayer finishes minelaying activity. Zero means no delay.")]
```

Comment:
```suggestion
		[Desc("Number of ticks for the minelayer to wait after laying a mine. The wait can be interrupted by a player order.")]
```

I think this sounds a bit better. IMO the zero value being instant is fairly self-explanatory (can also remove from the above statement)